telemetrylib/TCP.cpp

```diff
@@ -62,14 +66,38 @@ public slots:
         {
             QTcpSocket* sender = static_cast<QTcpSocket*>(QObject::sender());
             _sockets.removeOne(sender);
-            connected = false;
+            connection = false;
             emit connectionStatusChanged();
         }
     }
 private:
-    QTcpServer _server;     //the TCP socket
-    QList<QTcpSocket*> _sockets;    //The list of TCP socket
-    bool connected = false;     //This is used when getConnectionStatus() is called
+    QTcpServer _server;
+    QList<QTcpSocket*> _sockets;
+    std::atomic<bool> connection = false;
+    std::thread *t;
+    std::atomic<bool> isConnected=false;
+
+    /**
+     * creates a thread that pings the Chase car's static IP to check connection
+     */
+    void checkConnection() {
+        QTcpSocket sock;
+        while(!isConnected) {
+            sock.connectToHost("196.186.1.16", 80);
```

Comment:
```suggestion
            sock.connectToHost("196.168.1.16", 22);
```
Port 80 would refuse connection, Port 22 is should be open on distribution server

Reply:
Let's use port 4005 instead. I'd like to avoid using dedicated/special ports.